server/database/seeds/01_languages.js

```diff
@@ -1,13 +1,12 @@
-
 exports.seed = function(knex, Promise) {
   // Deletes ALL existing entries
-  return knex('table_name').del()
-    .then(function () {
+  return knex('languages')
+    .del()
+    .then(function() {
       // Inserts seed entries
-      return knex('table_name').insert([
-        {id: 1, colName: 'rowValue1'},
-        {id: 2, colName: 'rowValue2'},
-        {id: 3, colName: 'rowValue3'}
+      return knex('languages').insert([
+        { english_name: 'spanish', native_name: 'espanol' },
```

Comment:
probably need a seed for english too, since this is the table users will bounce off of to indicate their native language